applications/GeoMechanicsApplication/custom_elements/updated_lagrangian_U_Pw_diff_order_element.h

```diff
@@ -75,28 +74,20 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) UpdatedLagrangianUPwDiffOrderElement
     KRATOS_CLASS_INTRUSIVE_POINTER_DEFINITION(UpdatedLagrangianUPwDiffOrderElement);
 
     /// Default Constructor
-    UpdatedLagrangianUPwDiffOrderElement() : SmallStrainUPwDiffOrderElement() {}
+    UpdatedLagrangianUPwDiffOrderElement();

```

Comment:
I think this could be just an `=default` in the header:

```suggestion
    UpdatedLagrangianUPwDiffOrderElement() = default;
```

Reply:
done